plugins/Morpheus/templates/layout.twig

```diff
@@ -5,9 +5,8 @@
             <meta charset="utf-8">
             <title>
                 {%- block pageTitle -%}
-                    {%- if title is defined %}{{ title }} - {% endif %}
-                    {%- if categoryTitle is defined %}{{ categoryTitle }} - {% endif %}
-                    {%- if not isCustomLogo %}Piwik{% endif %}
+                    {% set branding = isCustomLogo ? '' : 'Piwik' %}
+                    {{ [title, categoryTitle|default, branding]|join(' - ')|trim('- ') }}
```

Comment:
I now see that there was before a check for `title is defined` so I'm now thinking the `title` might not be defined sometimes as well. To you mind adding a `title|default('')`? 

I'm wondering if it would actually work when title and category title is not defined? Wouldn't it be maybe something like ` - - Piwik` because it first starts with a whitespace and would be maybe not trimmed because of that? Maybe needs another `trim(' - ')` ?